packages/form-upload/src/FilePicker.js

```diff
@@ -33,7 +33,7 @@ const FilePicker = ({
     for (let i = 0; i < files.length; i++) {
       value[i] = files[i];
     }
-    setFieldValue(name, value, true);
+    // setFieldValue(name, value, true);
```

Comment:
Is this supposed to be commented out?

Reply:
Yup, supposed to be deleted. Will fix

Reply:
That was setting the context to a `File` instead of an `Upload` so I handle the update higher up